File: sanidad-backend/src/main/java/com/farmacia/sanidadbackend/dto/DevolucionDetalleResponse.java
package com.farmacia.sanidadbackend.dto;

import lombok.Data;
import java.math.BigDecimal;

@Data
public class DevolucionDetalleResponse {

    private Long id;
    private Long loteDetalleId;
    private String medicamentoNombre;
    private String loteNumero;
    private Integer cantidadDevuelta;
    private BigDecimal precioUnitario;
    private BigDecimal subtotal;

}

File: SanidadMovil/app/src/main/java/com/sanidad/movil/data/remote/dto/DevolucionDetalleResponse.kt
package com.sanidad.movil.data.remote.dto

data class DevolucionDetalleResponse(
    val id: Long,
    val loteDetalleId: Long,
    val medicamentoNombre: String,
    val loteNumero: String,
    val cantidadDevuelta: Int,
    val precioUnitario: Double,
    val subtotal: Double
)